common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -1070,14 +1069,44 @@ div.problem {
     &.warning {
       @include notification-by-type($warning-color);
     }
+    &.hint {
+      border: 1px solid $uxpl-gray-background;
+      border-radius: 7px;
```

Comment:
This block can go away since this was originally for Hint:

&.info {
   @include notification-by-type($info-color);
}